packages/terra-date-picker/src/DatePicker.jsx

```diff
@@ -468,6 +475,9 @@ class DatePicker extends React.Component {
                 onButtonFocus={this.handleFocus}
                 buttonRefCallback={(buttonRef) => { this.calendarButton = buttonRef; }}
                 ariaLabel={ariaLabel}
+                inputRefCallback={(inputRef) => {
```

Comment:
Can we pass this directly?
```suggestion
                inputRefCallback={inputRefCallback}
```

Reply:
Oh my bad i missed this, I had other expressions here before but since those have been removed i have changed this to pass `inputRefCallback` directly